resources/js/layouts/LandingLayout.vue

```
@@ -453,6 +453,7 @@ export default {
     },
 
     mounted() {
+    },
```

Biome (2.4.16)

[error] This method is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)